### PR TITLE
UIDEXP-81: Job profiles list - integration with backend

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**
## Description
Implement integration job profiles list with backend

## Issue
[UIDEXP-81](https://issues.folio.org/browse/UIDEXP-81)

## Screenshot
image

### CODE REVIEW

**Comment by YevheniiMaltsev** [comment]
@Vladyslav-Velytskyi Please, use local linter or add reviewers after handling lint and testing issues. Yes, there are some tests errors and we have to rerun the job. But this is usually a special case.

File: test/bigtest/network/scenarios/fetch-job-profiles-success.js [hunk 1]
```diff
@@ -1,53 +1,28 @@
...
+      updatedByUserId: 'dee12548-9cee-45fa-bbae-675c1cc0ce3b',
+      updatedByUsername: ''
+    }
+  });
 
-  server.get('/data-export/jobExecutions', (schema, request) => {
-    const { url } = request;
-    const statuses = url.match(/status=\((?<statuses>.*)\)/).groups.statuses.split(' OR ');
+  server.get('/data-export/jobProfiles', schema => {
+    const jobProfiles = schema.jobProfiles.all();
```
Comment (VictorSoroka1) [inline test/bigtest/network/scenarios/fetch-job-profiles-success.js:24]:
```suggestion
    retrurn schema.jobProfiles.all();
```

File: test/bigtest/network/scenarios/fetch-job-profiles-success.js [hunk 2]
```diff
@@ -1,53 +1,28 @@
...
+    mappingProfileId: '25d81cbe-9686-11ea-bb37-0242ac130002',
+    metadata: {
+      createdDate: '2018-12-04T11:22:07Z',
+      createdByUserId: 'dee12548-9cee-45fa-bbae-675c1cc0ce3b',
+      createdByUsername: 'janedoeuser',
+      updatedDate: '2018-12-04T13:28:54Z',
+      updatedByUserId: 'dee12548-9cee-45fa-bbae-675c1cc0ce3b',
+      updatedByUsername: ''
+    }
+  });
```
Comment (VictorSoroka1) [inline test/bigtest/network/scenarios/fetch-job-profiles-success.js:21]: Let's add one more job profile generated by the factory so we have multiple items in the list.

**Comment by VictorSoroka1** [comment]
Please replace `NoValue` with empty string [here](https://github.com/folio-org/ui-data-export/blob/master/src/settings/JobProfiles/JobProfilesContainer/JobProfilesContainer.js#L58).

**Comment by VictorSoroka1** [comment]
@YevheniiMaltsev found out that it already occurs two requests upon the page load for mapping profiles. I presume the same is true for job profiles as well. Please check this aspect as part of this task.

File: test/bigtest/network/scenarios/fetch-job-profiles-success.js (hunk 2 above)
Comment (Vladyslav-Velytskyi) [reply test/bigtest/network/scenarios/fetch-job-profiles-success.js:21]: Done

File: test/bigtest/network/scenarios/fetch-job-profiles-success.js (hunk 1 above)
Comment (Vladyslav-Velytskyi) [reply test/bigtest/network/scenarios/fetch-job-profiles-success.js]: Done

**Comment by YevheniiMaltsev** [comment]
@VictorSoroka1 it seems, we should add the permission "data-export.job-profiles.collection.get" to package json in scope of this story. What do you think about it?

**Comment by VictorSoroka1** [comment]
Completely agree, I saw that you added in your PR for mapping profiles which Vlad should have added as part of that integration. We should be careful with permissions stuff. So Vlad please address this aspect as well.

**Comment by Vladyslav-Velytskyi** [comment]
> Completely agree, I saw that you added in your PR for mapping profiles which Vlad should have added as part of that integration. We should be careful with permissions stuff. So Vlad please address this aspect as well.

Thanks for this comment, I added it, please double-check.

**Pull request merged**